Tasks/app-store-release/app-store-release.ts

```diff
@@ -81,7 +81,7 @@ async function run() {
             }
         }
 
-        let ipaPath: string = tl.getInput('ipaPath', true);
```

Comment:
Changing input name is a breaking change, so please leave the input name as ipaPath. Changing the variable to filePath is fine.